content/en/docs/Concepts/crds/bundleIndexImage.md

```diff
@@ -0,0 +1,97 @@
+
+Another OLM concept is the "index image", which is like a catalog of Operators. This container image acts to serve an application programming interface (API) which describes information about your sample operator and others in the catalog. An index image, based on the Operator Bundle Format, is a containerized snapshot of a catalog. It is an immutable artifact that contains the database of pointers to a set of Operator manifest content. A catalog can reference an index image to source its content for OLM on the cluster. 
+
+#### Creating an index image:
+
+
+1. The index image includes information from your bundle image by running the operator-registry command by using the opm command as follows:
+```
+opm index add --bundles <registry>/<username>/sample-operator-bundle:vX.X.X --tag <registry>/<username>/sample-operator-index:vX.X.X --binary-image <registry_base_image>
+```
+- A comma seperated list of additional bundle images to the index
+- The image tag that you want the index image to have.
+- Optional: An alternative registry base image to use for serving the catalog.
+
+2. Push the index image to a registry:
+```
+docker push <registry>/<username>/sample-operator-index:vX.X.X
+```
+
+The index image holds a sqlite database with bundle definitions, it also runs a grpc service when the image is executed. The grpc service lets consumers query the sqlite database for information about the operators this index contains.
+
+You can download the opm command from the [operator-registry](https://github.com/operator-framework/operator-registry) repository.
+
+#### Creating a catalog from an index image
+
+Create a CatalogSource object that references your index image. Modify the following to your specifications and save it as a catalogSource.yaml file:
+
+```yaml
+kind: CatalogSource
+metadata:
+  name: sample-operator
+  namespace: operators
+spec:
+  sourceType: grpc
+  image: <registry>:<port>/<namespace>/sample-operator-index:vX.X.X
+  displayName: My Operator Catalog
+  publisher: <publisher_name> 
+  updateStrategy:
+    registryPoll: 
+      interval: 30m
+```
+
+- Specify your index image
+- Specify your name or an organization name publishing the catalog
+- Catalog sources can automatically check for new versions to keep up to date
+
+Now we have to apply this yaml file to the kubernetes cluster:
+
+```
+kubectl apply -f catalogSource.yaml
+```
+
+This CatalogSource is created in an existing namespace by OLM named operators on OpenShift clusters. This namespace is called operators. When you create the CatalogSource, it causes the index image to be executed as a Pod. You can view it as follows:
+```
+kubectl -n operators get pod --selector=olm.catalogSource=sample-operator
+```
+
+#### Updating an index image
+
+After configuring OperatorHub to use a catalog source that references a custom index image that cluster administrators can keep the available Operators on their cluster up to date by adding bundle images to the index image.
```

Comment:
Did you mean OperatorHub? Nowhere else do you mention OperatorHub. So did the steps we executed put our image in OperatorHub? This statement is confusing.

Reply:
How would I configure OperatorHub? There should be a link to the document for this.